### DISCUSSION TITLE
how to add to add discussions - a discussions-board.

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#36156] [Discussions]
hello dear Friends, 
i have a github site with repos -  and a github-page. 

now i want to add a wikipage and discussions - a discussions-board.  Note - the wiki is allready set up - but the discussions not yet. 
is the setup  to find in the settings!`?

the few questions that come in mind: 
how to get started ?
what are the best practices to customize the discussions!?

Look forward to hear from you

**Comment by FahdSeddik** [comment]
Hey @fsj-digital 
You might find what you're looking for [here](https://docs.github.com/en/discussions/quickstart). Hope this helps!

**Comment by jfullstackdev** [comment]
at the repo Settings, scroll down, you should be seeing this 

![image](https://user-images.githubusercontent.com/47092464/196042970-0449c4a1-e62b-4db2-b731-5f11638a6571.png)

but I prefer doing it in an Org, you can always create Org without members, because the Discussion can be seen on the landing page of the Org, it's more presentable as for me
